config.json

```diff
@@ -158,5 +158,19 @@
             "bucket": "opentoken-io-test-s3",
             "region": "us-east-1"
         }
+    },
+    "token": {
+        "idHash": {
+            "algorithm": "sha512",
+            "derivedLength": 32,
+            "encoding": "base64-uri",
+            "iterations": 4000,
```

Comment:
This is the hash for mapping the incoming token to a different identifier.

As far as PBKDF2 goes, there doesn't seem to be any solid standard on the number of iterations.  Different sources recommend different levels.

I'm updating all of the iterations to be larger values in case they are < 10,000.